static/src/js/pages/KeywordManagerPage/__tests__/KeywordManagerPage.test.jsx

```diff
@@ -20,6 +26,28 @@ vi.mock('@/js/components/PageHeader/PageHeader', () => ({
   default: ({ title }) => <h1 data-testid="page-header">{title}</h1>
 }))
 
+vi.mock('@/js/components/KmsConceptVersionSelector/KmsConceptVersionSelector', () => ({
+  default: ({ onVersionSelect }) => (
+    <select
+      data-testid="version-selector"
+      onChange={
+        (e) => {
+          const [version, type] = e.target.value.split('|')
+          onVersionSelect({
+            version,
+            version_type: type
+          })
+        }
+      }
+    >
+      <option value="">Select a version</option>
+      <option value="1.0|draft">Version 1.0 (Draft)</option>
+      <option value="2.0|draft">Version 2.0 (Draft)</option>
+      <option value="3.0|published">Version 3.0 (Published)</option>
```

Comment:
instead of mocking the component, why not just mock the fetch, so you can test it from the fetch to what it being rendered in the combo?